.pylintrc

```diff
@@ -1,5 +1,5 @@
 [MESSAGE CONTROL]
-disable = abstract-method,missing-docstring,redefined-outer-name,too-few-public-methods
+disable = abstract-method,missing-docstring,redefined-outer-name,too-few-public-methods, too-many-arguments
```

Comment:
I guess that's caused by `oauth2.create_blueprint()`? Is there are better way to do DI?

Reply:
Yes it was. You could wrap some of them up but the `create_blueprint` is creating the blueprint with x3 view functions and x3 utility functions so its having to accept args in turn for them all. If it was a 'normal' non factory function I would say reduce it, but I don't think its unacceptable to have 6 arguments for a function doing this much.

That said I normally follow fat models, thin views, so any business logic that deals with finding or updating a model stays in the model class (or possibly the `repository` in this case). That would leave the views a lot thinner and therefore probably require less arguments to initialise.

Reply:
Can be reverted now? (We might hit it later though.)

Reply:
Yes that can go now. Will remove.